src/macros.rs

```diff
@@ -327,3 +327,148 @@ macro_rules! template {
+///
+/// * `label_expression` -- An expression that returns a label that implements
+///   the `RenderOnce` trait (e.g. `String` or `&str`).
+///
+/// * `label_expression => cond_test` -- Conditionally include `label_expression` whenever `cond_test` is `true`.
+///   `cond_test` is an expression that returns either `true` or `false`.
+///
+/// This useful in generating style attribute as follows:
+///
+/// ```norun
+/// html! {
+///     div(style = labels_sep_by!(";"; "color: #000" => true, "font-weight: bold")) {
+///         : "This is a button"
+///     }
+/// }
+/// ```
+///
+#[macro_export]
+macro_rules! labels_sep_by {
+
+    (@inner_expand $has_before:expr; $sep:expr; $tmpl:ident $item:expr) => {
+        if $has_before {
+            $crate::RenderOnce::render_once($sep, $tmpl);
+        }
+        $crate::RenderOnce::render_once($item, $tmpl);
+    };
+
+    (@inner_expand $has_before:expr; $sep:expr; $tmpl:ident $item:expr => $should_include:expr) => {
+        if $should_include {
+            if $has_before {
+                $crate::RenderOnce::render_once($sep, $tmpl);
+            }
+            $crate::RenderOnce::render_once($item, $tmpl);
+        }
+    };
+
+    (@inner_expand $has_before:expr; $sep:expr; $tmpl:ident $item:expr, $($tail:tt)+) => {
+        if $has_before {
+            $crate::RenderOnce::render_once($sep, $tmpl);
+        }
+        $crate::RenderOnce::render_once($item, $tmpl);
+        labels_sep_by!(@inner_expand true; $sep; $tmpl $($tail)*);
+    };
+
+    (@inner_expand $has_before:expr; $sep:expr; $tmpl:ident $item:expr => $should_include:expr, $($tail:tt)+) => {
+        if $should_include {
+            if $has_before {
+                $crate::RenderOnce::render_once($sep, $tmpl);
+            }
+            $crate::RenderOnce::render_once($item, $tmpl);
+            labels_sep_by!(@inner_expand true; $sep; $tmpl $($tail)*);
+        } else {
+            labels_sep_by!(@inner_expand $has_before; $sep; $tmpl $($tail)*);
+        }
+
+    };
+
+    // entries
+
+    ($sep:expr; $item:expr) => {
+
+        $crate::FnRenderer::new(|tmpl| {
+            $crate::RenderOnce::render_once($item, tmpl);
+        })
+
+    };
+
+    ($sep:expr; $item:expr => $should_include:expr) => {
+
+        $crate::FnRenderer::new(|tmpl| {
+            if $should_include {
+                $crate::RenderOnce::render_once($item, tmpl);
+            }
+        })
+
+    };
+
+    ($sep:expr; $item:expr, $($tail:tt)+) => {
+
+        $crate::FnRenderer::new(|tmpl| {
+            $crate::RenderOnce::render_once($item, tmpl);
+            labels_sep_by!(@inner_expand true; $sep; tmpl $($tail)*);
+        })
+
+    };
+
+    ($sep:expr; $item:expr => $should_include:expr, $($tail:tt)+) => {
+
+        $crate::FnRenderer::new(|tmpl| {
+            if $should_include {
+                $crate::RenderOnce::render_once($item, tmpl);
+                labels_sep_by!(@inner_expand true; $sep; tmpl $($tail)*);
+            } else {
```

Comment:
Same here. You can just pass `$should_include` instead of the `true` and `false` constants.